app/assets/stylesheets/layout.scss

```diff
@@ -124,9 +130,10 @@ body {
   }
 
   .pagination {
-    display: block;
+    display: flex;
+    justify-content: center;
     max-width: none;
-    overflow-x: visible;
+    overflow-x: scroll;
```

Comment:
This looks good, but can we make it so that the scrollbar is hidden when it's not needed?